src/backend/app/Models/Meter/Meter.php

```diff
@@ -86,4 +87,8 @@ public function findBySerialNumber(string $meterSerialNumber): ?self {
     public function getId(): int {
         return $this->id;
     }
+
+    public function geo(): MorphOne {
+        return $this->morphOne(GeographicalInformation::class, 'owner');
```

Comment:
I don't think `Meter` is directly related to `GeoGraphicalInformation`. Instead it's `Address`<>`Device`<>`Meter`